src/bls12_381/fq.rs

```diff
@@ -42,6 +47,16 @@ const R2: FqRepr = FqRepr([
 // INV = -(q^{-1} mod 2^64) mod 2^64
 const INV: u64 = 0x89f3fffcfffcfffd;
 
+// SWENC_CONST0 = sqrt(-3) mod q =
```

Comment:
I think these constants should be named closer to what they are, because I can't remember. :)

`SWENC_SQRT_NEG_THREE` and `SWENC_SQRT_NEG_THREE_MINUS_ONE_OVER_TWO` sound good. (Yes, they're long, but that's better than being ambiguous. :)